Decred Wallet/view_controller/ReceiveViewController.swift

```diff
@@ -45,17 +48,15 @@ class ReceiveViewController: UIViewController,UIDocumentInteractionControllerDel
 
     override func viewDidLoad() {
         super.viewDidLoad()
-        self.subheader.text = LocalizedStrings.receiveHeaderInfo
-        // TAP Gesture
-        self.setupExtraUI()
+        
         self.starttime = Int64(NSDate().timeIntervalSince1970)
```

Comment:
Please remove this line and the `starttime` property. Don't know why they were added before, but they're useless now.